
diff -up knetworkmanager/knetworkmanager-0.7/src/knetworkmanager-connection_setting_wireless_security.cpp.fix knetworkmanager/knetworkmanager-0.7/src/knetworkmanager-connection_setting_wireless_security.cpp
--- knetworkmanager/knetworkmanager-0.7/src/knetworkmanager-connection_setting_wireless_security.cpp.fix	2009-03-02 15:28:38.000000000 -0500
+++ knetworkmanager/knetworkmanager-0.7/src/knetworkmanager-connection_setting_wireless_security.cpp	2009-03-02 15:30:01.000000000 -0500
@@ -290,14 +290,9 @@ WirelessSecurity::toMap() const
 			map.insert(NM_SETTING_WIRELESS_SECURITY_PAIRWISE, QDBusData::fromQValueList(ciphers));
 	}
 
-	if (!_psk.isNull())
-		map.insert(NM_SETTING_WIRELESS_SECURITY_PSK, QDBusData::fromString(""));
-
 	if (!_leapUsername.isEmpty())
 		map.insert(NM_SETTING_WIRELESS_SECURITY_LEAP_USERNAME, QDBusData::fromString(_leapUsername));
 
-	if (!_leapPassword.isEmpty())
-		map.insert(NM_SETTING_WIRELESS_SECURITY_LEAP_PASSWORD, QDBusData::fromString(""));
 	return map;
 }
 

