--- knetworkmanager-0.7.0_p20090414.orig/knetworkmanager-0.7/knetworkmanager.conf	2009-04-14 17:52:48.000000000 +0200
+++ knetworkmanager-0.7.0_p20090414/knetworkmanager-0.7/knetworkmanager.conf	2009-04-14 17:53:28.000000000 +0200
@@ -16,7 +16,7 @@
 		       send_interface="org.freedesktop.NetworkManagerSettings.Connection.Secrets"/>
 
 	</policy>
-	<policy at_console="true">
+	<policy group="plugdev">
 		<allow own="org.freedesktop.NetworkManagerUserSettings"/>
 
 		<allow send_destination="org.freedesktop.NetworkManagerUserSettings"
