From dd6280e158f85480820e23f9de0b214f0a17631d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lukas=20Sandstr=C3=B6m?= <luksan@gmail.com>
Date: Mon, 15 Feb 2010 21:47:46 +0100
Subject: [PATCH] fix import of empty entry

---
 kaddressbook/xxport/gnokii_xxport.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kaddressbook/xxport/gnokii_xxport.cpp b/kaddressbook/xxport/gnokii_xxport.cpp
index e78de40..6ce634c 100644
--- a/kaddressbook/xxport/gnokii_xxport.cpp
+++ b/kaddressbook/xxport/gnokii_xxport.cpp
@@ -299,7 +299,7 @@ static gn_error read_phone_entries( const char *memtypestr, gn_memory_type memty
 	progress->setValue(num_read);
   	this_filter->processEvents();
 
-	if (error == GN_ERR_EMPTYLOCATION)
+	if (error == GN_ERR_EMPTYLOCATION || entry.empty)
 		continue;
 	if (error == GN_ERR_INVALIDLOCATION)
 		break;
-- 
1.6.6.1

