--- kaddressbook-4.3.3/kaddressbook/xxport/gnokii_xxport.cpp.orig	2009-03-09 12:27:25.000000000 +0100
+++ kaddressbook-4.3.3/kaddressbook/xxport/gnokii_xxport.cpp	2009-03-08 21:26:49.000000000 +0100
@@ -116,7 +116,7 @@ from http://bugsfiles.kde.org/attachment.cgi?id=31947
 #endif
 		return i18n("Failed to initialize the gnokii library.");
 
-	if (!gn_cfg_phone_load("", &state))
+	if (gn_cfg_phone_load("", &state) != GN_ERR_NONE)
 		return i18n("Gnokii is not yet configured.");
 
 	// uncomment to debug all gnokii communication on stderr.
