--- parts/appwizard/common/admin/libtool.m4.in	2009-07-26 16:25:43.000000000 +0200
+++ parts/appwizard/common/admin/libtool.m4.in	2007-11-30 23:39:28.000000000 +0100
@@ -284,10 +284,10 @@
 # Check that we are running under the correct shell.
 SHELL=${CONFIG_SHELL-/bin/sh}
 
-case X$echo in
+case X$ECHO in
 X*--fallback-echo)
   # Remove one level of quotation (which was required for Make).
-  ECHO=`echo "$echo" | sed 's,\\\\\[$]\\[$]0,'[$]0','`
+  ECHO=`echo "$ECHO" | sed 's,\\\\\[$]\\[$]0,'[$]0','`
   ;;
 esac
 
@@ -319,7 +319,7 @@
 # if CDPATH is set.
 if test "X${CDPATH+set}" = Xset; then CDPATH=:; export CDPATH; fi
 
-if test -z "$echo"; then
+if test -z "$ECHO"; then
 if test "X${echo_test_string+set}" != Xset; then
 # find a string as large as possible, as long as the shell can cope with it
   for cmd in 'sed 50q "[$]0"' 'sed 20q "[$]0"' 'sed 10q "[$]0"' 'sed 2q "[$]0"' 'echo test'; do
@@ -423,7 +423,7 @@
 # Copy echo and quote the copy suitably for passing to libtool from
 # the Makefile, instead of quoting the original, which is used later.
 ECHO=$echo
-if test "X$echo" = "X$CONFIG_SHELL [$]0 --fallback-echo"; then
+if test "X$ECHO" = "X$CONFIG_SHELL [$]0 --fallback-echo"; then
    ECHO="$CONFIG_SHELL \\\$\[$]0 --fallback-echo"
 fi
 
